<?xml version="1.0"?>
<launch>
    <group ns="unav2">
        <rosparam file="$(find unav2_hardware)/config/unav.yaml" command="load" />
        <node pkg="unav2_hardware" type="hardware_node" name="hardware_node" output="screen" respawn="true" respawn_delay="2"/>
        <node pkg="rosserial_python" type="message_info_service.py" name="message_info_service" output="screen" respawn="true" respawn_delay="2"/>
        <node pkg="unav2_hardware" type="configuration_manager.py" name="unav2_configuration_manager" output="screen" respawn="true" respawn_delay="2"/>
    </group>
</launch>
